gammapy/spectrum/flux_point.py

```diff
@@ -584,13 +584,19 @@ class FluxPointEstimator(object):
         Energy groups (usually output of `~gammapy.spectrum.SpectrumEnergyGroupMaker`)
     model : `~gammapy.spectrum.models.SpectralModel`
         Global model (usually output of `~gammapy.spectrum.SpectrumFit`)
+    nsigma_ul_threshold : float
+        Significance for an energy bin for which an UL should be computed
+    nsigma_ul : float
+        Number of sigma used for the UL computation
     """
 
-    def __init__(self, obs, groups, model):
+    def __init__(self, obs, groups, model, nsigma_ul_threshold=1.5, nsigma_ul=3.):
```

Comment:
I think the default values should be changed here (see my comment above) and are best set only once.